src/borneo/http.py

```diff
@@ -567,8 +567,26 @@ def _process_ok_response(self, bis, request):
         exception.
         """
         err = serde.BinaryProtocol.read_string(bis)
+
+        # special case for TNF errors on WriteMultiple with many tables
+        if code == serde.BinaryProtocol.USER_ERROR.TABLE_NOT_FOUND and \
+                isinstance(request, operations.WriteMultipleRequest):
+            raise self._handle_write_multiple_table_not_found(code, err, request)
+
         raise serde.BinaryProtocol.map_exception(code, err)
 
+    @staticmethod
+    def _handle_write_multiple_table_not_found(code, err, wm_request):
```

Comment:
Please add a comment here, similar to the other SDKs, that explains why the ',' and '[' checks.

Reply:
Done